VikingQuiz.UI/src/Components/QuizItem/QuizItem.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import './QuizItem.css';
+import ManageQuizComponent from '../MyQuizesPage/ManageQuizComponent/ManageQuizComponent';
+
+
+class QuizItem extends React.Component<any, any>{
+    constructor(props: any) {
+        super(props);
+  
+        this.state = {
+            serverMessage: '',
+            redirect: false,
+            isHidden: true,
+            id: this.props.id,
```

Comment:
I'm surprised this works. Anyway, as said above, if you're not going to update id, don't keep it as a state, just pass it along as props